conda_build/conda_interface.py

```diff
@@ -131,24 +101,15 @@ def try_exports(module, attr):
 create_default_packages = context.create_default_packages
 
 get_rc_urls = lambda: list(context.channels)
-get_prefix = partial(context_get_prefix, context)
+get_prefix = partial(context_get_prefix, context)  # noqa: F811, F821
```

Comment:
neither noqa is needed with the fix above:

```suggestion
get_prefix = partial(context_get_prefix, context)
```